packages/serverless/src/apollo/app/collection.ts

```diff
@@ -108,6 +112,17 @@ const collectionCreate = authorizeWrapper(
         session
       )
     );
+
+    if (createCollectionResult) {
+      await createIndex(
```

Comment:
why dont you put createIndex inside the transaction block

Reply:
It appears that when I create an index inside the transaction block, the same transaction ends up calling createCollection (domain) twice

Reply:
That's because you didn't put session(tx) into createIndex. How about when createCollection success but createIndex not success?

Reply:
`createIndex` does not work within session

Reply:
So if `createIndex` fail. createCollectionResult will continue as usual without createIndex. Is it normal?

Reply:
The main operation was done, if fail we can send the error

Reply:
if error send. the collection still create and user have to createIndex manually ?